From 46c0789cf37fcd04a536127fd12f892d13ddcc0a Mon Sep 17 00:00:00 2001
From: Romain Francoise <rfrancoise@debian.org>
Date: Thu, 24 Feb 2011 23:26:08 +0100
Subject: [PATCH 5/7] (describe-copying) uses /usr/share/common-licenses/GPL

On Debian systems, the full text of the GPL is available from the
/usr/share/common-licenses directory.  The COPYING file included in
the Emacs distribution is not installed in `data-directory', so
`describe-copying' has been modified accordingly.

Signed-off-by: Julien Danjou <julien@danjou.info>
---
 lisp/help.el | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lisp/help.el b/lisp/help.el
index e80d2d8..38c5b08 100644
--- a/lisp/help.el
+++ b/lisp/help.el
@@ -292,7 +292,8 @@ If that doesn't give a function, return nil."
 (defun describe-copying ()
   "Display info on how you may redistribute copies of GNU Emacs."
   (interactive)
-  (view-help-file "COPYING"))
+  (view-file "/usr/share/common-licenses/GPL")
+  (goto-char (point-min)))
 
 (defun describe-gnu-project ()
   "Display info on the GNU project."
-- 
1.8.5.2

